sources/packages/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -4,6 +4,9 @@ import { BCeIDDetailsDto } from "../../../route-controllers/user/models/bceid-ac
 import { Institution } from "../../../database/entities";
 
 export class CreateInstitutionDto {
+  @IsNotEmpty()
```

Comment:
During institution creation, userEmail is not getting mapped because CreateInstitutionDto  does not have it.

And while saving to DB, primaryUserEmail is mapped to userEmail which was causing the bug.